### PR TITLE
HDDS-1198. Rename chill mode to safe mode. Contributed by Siddharth Wagle.

### PR DESCRIPTION
**Pull request by arp7**

### CODE REVIEW

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java [hunk 1]
```diff
@@ -23,12 +23,12 @@
 import org.apache.hadoop.hdds.protocol.proto.StorageContainerLocationProtocolProtos.ListPipelineRequestProto;
 import org.apache.hadoop.hdds.protocol.proto.StorageContainerLocationProtocolProtos.ListPipelineResponseProto;
 import org.apache.hadoop.hdds.protocol.proto.StorageContainerLocationProtocolProtos.ClosePipelineRequestProto;
-import org.apache.hadoop.hdds.protocol.proto.StorageContainerLocationProtocolProtos.ForceExitChillModeRequestProto;
-import org.apache.hadoop.hdds.protocol.proto.StorageContainerLocationProtocolProtos.ForceExitChillModeResponseProto;
```
Comment (bharatviswa504) [inline hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java]: Line length greater than 80.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 21 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 26 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 76 | Maven dependency ordering for branch |
| +1 | mvninstall | 1183 | trunk passed |
| +1 | compile | 2225 | trunk passed |
| +1 | checkstyle | 419 | trunk passed |
| -1 | mvnsite | 83 | integration-test in trunk failed. |
| +1 | shadedclient | 2211 | branch has no errors when building and testing our client artifacts. |
| 0 | findbugs | 0 | Skipped patched modules with no Java source: hadoop-ozone/integration-test |
| +1 | findbugs | 843 | trunk passed |
| +1 | javadoc | 1193 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 106 | Maven dependency ordering for patch |
| -1 | mvninstall | 62 | client in the patch failed. |
| +1 | compile | 1545 | the patch passed |
| +1 | cc | 1545 | the patch passed |
| +1 | javac | 1545 | the patch passed |
| +1 | checkstyle | 210 | the patch passed |
| +1 | mvnsite | 300 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 1 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 726 | patch has no errors when building and testing our client artifacts. |
| 0 | findbugs | 0 | Skipped patched modules with no Java source: hadoop-ozone/integration-test |
| +1 | findbugs | 426 | the patch passed |
| +1 | javadoc | 252 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 31 | client in the patch passed. |
| +1 | unit | 90 | common in the patch passed. |
| -1 | unit | 100 | server-scm in the patch failed. |
| +1 | unit | 30 | tools in the patch passed. |
| +1 | unit | 45 | common in the patch passed. |
| -1 | unit | 846 | integration-test in the patch failed. |
| +1 | unit | 53 | ozone-manager in the patch passed. |
| +1 | unit | 88 | tools in the patch passed. |
| +1 | asflicense | 48 | The patch does not generate ASF License warnings. |
| | | 13802 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdds.scm.server.TestSCMClientProtocolServer |
|   | hadoop.ozone.web.client.TestKeysRatis |
|   | hadoop.ozone.TestMiniChaosOzoneCluster |
|   | hadoop.ozone.container.ozoneimpl.TestOzoneContainer |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-737/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/737 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  cc  xml  |
| uname | Linux 8112f4ef17e6 4.4.0-143-generic #169~14.04.2-Ubuntu SMP Wed Feb 13 15:00:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 62f4808 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| mvnsite | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/1/artifact/out/branch-mvnsite-hadoop-ozone_integration-test.txt |
| findbugs | v3.1.0-RC1 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/1/artifact/out/patch-mvninstall-hadoop-hdds_client.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/1/artifact/out/patch-unit-hadoop-hdds_server-scm.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/1/artifact/out/patch-unit-hadoop-ozone_integration-test.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/1/testReport/ |
| Max. process+thread count | 4328 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/client hadoop-hdds/common hadoop-hdds/server-scm hadoop-hdds/tools hadoop-ozone/common hadoop-ozone/integration-test hadoop-ozone/ozone-manager hadoop-ozone/tools U: . |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by swagle** [comment]
Manually ran and verified all of the above tests, following test is an actual issue and updating patch with a fix:
hadoop.hdds.scm.server.TestSCMClientProtocolServer

**Comment by arp7** [comment]
Looks like the anzix CI run did not get triggered. Reapplied label to trigger it.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 93 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 26 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 40 | Maven dependency ordering for branch |
| -1 | mvninstall | 39 | root in trunk failed. |
| -1 | compile | 55 | root in trunk failed. |
| +1 | checkstyle | 234 | trunk passed |
| -1 | mvnsite | 42 | integration-test in trunk failed. |
| +1 | shadedclient | 1320 | branch has no errors when building and testing our client artifacts. |
| 0 | findbugs | 0 | Skipped patched modules with no Java source: hadoop-ozone/integration-test |
| +1 | findbugs | 307 | trunk passed |
| +1 | javadoc | 175 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 24 | Maven dependency ordering for patch |
| -1 | mvninstall | 19 | client in the patch failed. |
| +1 | compile | 1288 | the patch passed |
| -1 | cc | 1288 | root generated 9 new + 0 unchanged - 0 fixed = 9 total (was 0) |
| -1 | javac | 1288 | root generated 1409 new + 87 unchanged - 0 fixed = 1496 total (was 87) |
| +1 | checkstyle | 203 | the patch passed |
| +1 | mvnsite | 296 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 1 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 669 | patch has no errors when building and testing our client artifacts. |
| 0 | findbugs | 0 | Skipped patched modules with no Java source: hadoop-ozone/integration-test |
| +1 | findbugs | 423 | the patch passed |
| +1 | javadoc | 243 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 32 | client in the patch passed. |
| +1 | unit | 117 | common in the patch passed. |
| +1 | unit | 140 | server-scm in the patch passed. |
| +1 | unit | 30 | tools in the patch passed. |
| +1 | unit | 45 | common in the patch passed. |
| -1 | unit | 937 | integration-test in the patch failed. |
| +1 | unit | 80 | ozone-manager in the patch passed. |
| +1 | unit | 180 | tools in the patch passed. |
| +1 | asflicense | 55 | The patch does not generate ASF License warnings. |
| | | 7313 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdds.scm.pipeline.TestNodeFailure |
|   | hadoop.hdds.scm.pipeline.TestNode2PipelineMap |
|   | hadoop.hdds.scm.safemode.TestSCMSafeModeWithPipelineRules |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/737 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  cc  xml  |
| uname | Linux a6d78e13341a 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 1943db5 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/artifact/out/branch-mvninstall-root.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/artifact/out/branch-compile-root.txt |
| mvnsite | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/artifact/out/branch-mvnsite-hadoop-ozone_integration-test.txt |
| findbugs | v3.1.0-RC1 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/artifact/out/patch-mvninstall-hadoop-hdds_client.txt |
| cc | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/artifact/out/diff-compile-cc-root.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/artifact/out/diff-compile-javac-root.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/artifact/out/patch-unit-hadoop-ozone_integration-test.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/testReport/ |
| Max. process+thread count | 3078 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/client hadoop-hdds/common hadoop-hdds/server-scm hadoop-hdds/tools hadoop-ozone/common hadoop-ozone/integration-test hadoop-ozone/ozone-manager hadoop-ozone/tools U: . |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-737/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by swagle** [comment]
@arp7: The 3 tests failures seem to be due to OOM.

**Pull request merged**